pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `if`                    | Core          | 07                 | No      | No                             |
+| `items` (legacy)        | Core          | 01                 | 2020-12 | Replaced by `prefixItems`                       |
+| `items`                 | Core          | 2020-12            | No      | Replaced `additionalItems` and `items` (legacy) |
+| `maxContains`           | Validation    | 2019-09            | No      | No                             |
+| `maxProperties`         | Validation    | 04                 | No      | No                             |
+| `maximumCanEqual`       | Validation    | 01                 | 03      | Replaced by `exclusiveMaximum` |
+| `minimumCanEqual`       | Validation    | 01                 | 03      | Replaced by `exclusiveMinimum` |
+| `minContains`           | Validation    | 2019-09            | No      | No                             |
+| `minProperties`         | Validation    | 04                 | No      | No                             |
+| `multipleOf`            | Validation    | 04                 | No      | Replaced `divisibleBy`         |
+| `not`                   | Core          | 04                 | No      | No                             |
+| `oneOf`                 | Core          | 04                 | No      | No                             |
+| `optional`              | Validation    | 02                 | No      | Replaced by `required`         |
+| `pattern`               | Core          | 03                 | No      | No                             |
+| `patternProperties`     | Core          | 03                 | No      | No                             |
+| `prefixItems`           | Core          | 2020-12            | No      | Replaced `items`               |
+| `propertyNames`         | Core          | 06                 | No      | No                             |
+| `readOnly`              | Validation    | 01                 | No      | No                             |
+| `required`              | Validation    | 03                 | No      | No                             |
+| `requires`              | Core          | 01                 | 03      | Replaced by `dependencies`     |
+| `title`                 | Validation    | 01                 | No      | No                             |
+| `then`                  | Core          | 07                 | No      | No                             |
+| `type` (legacy)         | Core          | 01                 | 04      | Replaced by `type`             |
+| `type`                  | Validation    | 04                 | No      | Replaced `type` (legacy)       |
+| `unevaluatedItems`      | Core          | 2019-09            | No      | No                             |
+| `unevaluatedProperties` | Core          | 2019-09            | No      | No                             |
+| `uniqueItems`           | Core          | 02                 | Yes     | No                             |
```

Comment:
I think its even more complicated, as for keywords like this one, older specifications had it in the "Core" one but now they are on "Validation". Given that, maybe we should completely drop the "Specification" column?

Reply:
Ah, I see you often do have multiple entries with different specifications, like with `type`

Reply:
@jviotti 
We are still going to have individual pages for the migration. This is meant to give an overview, and somehow, creating an individual page defeats the purpose of an overview. 

We can make it work as it is. We only need to get accurate information.

Reply:
> older specifications had it in the "Core" one but now they are on "Validation"

What I was trying to do with this in my suggestions was to use the specification that it last appeared in. I may have messed up some of my suggestions or just missed some, but that's what I was trying to do. So, `uniqueItems` would be in `Validation`. If you have a better way to communicate this, please do that instead.

Reply:
> We are still going to have individual pages for the migration.

If this table is just an overview and there's going to be somewhere else where some of these complicated situations appear, it wouldn't be too bad to oversimplify some of the more complicated situations here. Again, it's up to you.

Reply:
For the overview page, like in the case of `uniqueItems`, I will stick to the current specification it belongs to - `validation`. In Draft 2, we can add details of its change in the details section.